evap/evaluation/models.py

```diff
@@ -700,6 +700,13 @@ def reviewed_textanswer_set(self):
     def num_reviewed_textanswers(self):
         return self.reviewed_textanswer_set.count()
 
+    @property
+    def text_answer_review_is_urgent(self):
+        if self.state != "evaluated":
+            return False
+
+        return self.course.final_grade_documents or self.course.gets_no_grade_documents or not self.wait_for_grade_upload_before_publishing
```

Comment:
Should I add some linebreaks?

Reply:
I would prefer them :)
Also, while you are at breaking up the line, could you experiment with replacing the `if` with a `self.state == "evaluated" and …`? But I also didn’t come up with a nice solution

Reply:
I thought about it a bit, and for me, using the guard-clause is easier to read than the big boolean expression. I'll assume leaving that as it is now is fine for you(?)